### PR TITLE
Updated README with basic CMS docs

### LINKED ISSUE #486: create docs for adding new users to WagTail

### PR DESCRIPTION
**Pull request by pdpinch**
#### What are the relevant tickets?

fixes #486 

#### What's this PR do?

Starts a documentation section for the Wagtail CMS.

#### How should this be manually tested?

Try to follow the directions for creating a new program page and granting permission to edit it. 

#### Any background context you want to provide?

Let me know if you think a separate documentation page is justified. 

Creating CMS users and granting permission is pretty basic at this point. I added some stories (#685, #687) to make it more sophisticated. If those are worked on, we will need to update these docs.   

#### What GIF best describes this PR or how it makes you feel?

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,6 +191,36 @@ You should now be able to do the following:
...
+#### Adding CMS users
+
+1. Login to the cms with an existing account. If you don't have one, you can use the superuser account created earlier.
+
+2. From the Settings menu in the left nav, choose users
+
+3. You can use the search box to locate users who already exist. If the user already exists, click on the username and 
+skip to step 5.
+
+4. Don't create new users from the CMS. Ask users to log in and fill out a MicroMasters profile first.
```
Comment (giocalitri) [inline README.md]: I would put this as first bullet

**Comment by giocalitri** [comment]
just a tiny comment (that can be ignored). Other than that 👍

File: README.md [hunk 2]
```diff
@@ -191,6 +191,36 @@ You should now be able to do the following:
...
+
+2. Click on the `Explorer` menu in the left nav to find the home page (labelled "MIT credentials are...")
+
+3. Click on `+ Add Child Page`
+
+4. Choose Program Page. Complete the form. Don't forget to save. 
+
+#### Adding CMS users
+
+1. Don't create new users from the CMS. Ask users to log in and fill out a MicroMasters profile first.
```
Comment (pdpinch) [inline README.md]: @giocalitri is this what you meant?

Comment (giocalitri) [inline README.md]: yes!

**Pull request merged**